tools/src/main/scala/scala/scalanative/interflow/Eval.scala

```diff
@@ -90,6 +90,7 @@ trait Eval { self: Interflow =>
             throw BailOut("try-catch")
           }
           return Inst.Unreachable(Next.None)
+        case _ => BailOut
```

Comment:
I'm pretty sure this does nothing but loading the companion object `BailOut`. It should be `bailOut` to call the `def` defined above. Also, since all the other `case`s in this match have their bodies on separate lines, consider following the style, as follows:
```suggestion
        case _ =>
          bailOut
```

Reply:
Good catch!  Thank you.   Implemented as suggested.